server/src/test/java/io/spine/server/BoundedContextTest.java

```diff
@@ -38,11 +44,19 @@
 import io.spine.test.bc.Project;
 import io.spine.test.bc.SecretProject;
 import io.spine.testing.server.model.ModelTests;
+import javafx.util.Pair;
```

Comment:
Let's avoid using tuple types of any kind. If JUnit does not provide a convenient API to link two values, please define a custom class for that.